client/cgo/cgotest.go

```diff
@@ -0,0 +1,222 @@
+package main
+
+/*
+int testVerifyVrf(unsigned char *pk, int pkSize,
+    unsigned char *m, int mSize,
+    unsigned char *index, int indexSize,
+    unsigned char *proof, int proofSize) {
+    return cgoVerifyVrf(pk, pkSize, m, mSize, index, indexSize, proof, proofSize);
+}
+
+int testVerifySignature(
+    unsigned char *pk, int pkSize,
+    unsigned char *m, int mSize,
+    unsigned char *sig, int sigSize) {
+    return cgoVerifySignature(pk, pkSize, m, mSize, sig, sigSize);
+}
+
+int testHashChainInvalidSize() {
```

Comment:
Yup, my mistake. I was coding this why reading the old version of the paper and definitely had some misunderstanding. I'm going to fix it now. Thanks a lot!